clc/modules/object-storage/src/main/java/com/eucalyptus/objectstorage/ObjectStorageGateway.java

```diff
@@ -475,6 +492,7 @@ protected PutObjectResponseType doPutOperation(final PutObjectType request) thro
       } catch (Exception ex) {
         LOG.debug("Failed to fire reporting event for OSG object creation", ex);
       }
+      OSGUtil.setCorsInfo(request, response, bucket.getBucketUuid());
```

Comment:
Why is CORS information stuffed into responses for a few operations (delete, put, get, head, list bucket) and not others?